test/e2e_flags.go

```diff
@@ -68,7 +70,13 @@ func SetupLoggingFlags() {
 
 // ImagePath is a helper function to transform an image name into an image reference that can be pulled.
 func ImagePath(name string) string {
-	tpl, err := template.New("image").Parse(Flags.ImageTemplate)
+	tpl, err := template.New("image").Funcs(template.FuncMap{
+		"toEnvKey": func(in string) string {
+			underscores := strings.ReplaceAll(in, "-", "_")
+			return strings.ToUpper(underscores)
+		},
+		"fromEnv": os.Getenv,
```

Comment:
could use `os.ExpandEnv` here which would avoid needing the printf in the template